libraries/nginx_service.rb

```diff
@@ -94,6 +96,7 @@ def action_enable
       directory '/run/nginx' do
```

Comment:
I imagine that this doesn't exist on EL6 - we are fine with creating this directory?